meta-mbl-distro/classes/mbl-partitions.bbclass

```diff
@@ -346,9 +374,13 @@ python __anonymous() {
             bb.fatal("{} is not set".format(offset_var))
         elif have_prev:
             offset = _calculate_next_offset(align, prev_offset, prev_size)
+            if prev_fs_part_number > 2:
+                # We need to leave space for an alignment sized chunk for a
+                # 512B extended boot record
+                offset = _calculate_next_offset(align, offset, 1)
```

Comment:
I don't understand why the `prev_size` is 1.

Reply:
https://github.com/ARMmbed/meta-mbl/pull/645/commits/c3be70492abbee4850ceaaef4de7850539d7cc6d